ogticrd/covid-safe-paths
# Fix: Refactoring data list

#### Description:
Deleting a duplicate DataList file located on ActionCards folder


#### Linked issues:


Fix: [254](https://trello.com/c/XNlQq6Tr/245-as-a-developer-i-want-to-refactor-the-code-so-that-i-can-find-duplicate-code-3)

#### How to test:


- `git pull && git checkout Fix/245-refactoring-dataList`
- Head over News screen and all the news should work as expected
